### ISSUE TITLE
[Bug] be will core dump when load data 

### ISSUE THREAD

**Issue by yangzhg** [apache/doris#12410]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

be will core dump when load data 

```
*** Aborted at 1662525804 (unix time) try "date -d @1662525804" if you are using GNU date ***
*** Current BE git commitID: 46776af2a ***
*** SIGSEGV unkown detail explain (@0x0) received by PID 57618 (TID 0x7fa52b743700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/common/signal_handler.h:420
 1# 0x00007FA584F25570 in /lib64/libc.so.6
 2# doris::vectorized::ColumnNullable::insert_from(doris::vectorized::IColumn const&, unsigned long) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/columns/column_nullable.cpp:220
 3# void doris::vectorized::IColumn::append_data_by_selector_impl<doris::vectorized::ColumnNullable>(COW<doris::vectorized::IColumn>::mutable_ptr<doris::vectorized::IColumn>&, doris::vectorized::PODArray<unsigned long, 4096ul, Allocator<false, false>, 15ul, 16ul> const&) const at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/columns/column_impl.h:73
 4# doris::vectorized::ColumnNullable::append_data_by_selector(COW<doris::vectorized::IColumn>::mutable_ptr<doris::vectorized::IColumn>&, doris::vectorized::PODArray<unsigned long, 4096ul, Allocator<false, false>, 15ul, 16ul> const&) const at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/columns/column_nullable.h:173
 5# doris::vectorized::Block::append_block_by_selector(std::vector<COW<doris::vectorized::IColumn>::mutable_ptr<doris::vectorized::IColumn>, std::allocator<COW<doris::vectorized::IColumn>::mutable_ptr<doris::vectorized::IColumn> > >&, doris::vectorized::PODArray<unsigned long, 4096ul, Allocator<false, false>, 15ul, 16ul> const&) const at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/core/block.cpp:644
 6# doris::stream_load::VNodeChannel::add_block(doris::vectorized::Block*, std::pair<std::unique_ptr<doris::vectorized::PODArray<unsigned long, 4096ul, Allocator<false, false>, 15ul, 16ul>, std::default_delete<doris::vectorized::PODArray<unsigned long, 4096ul, Allocator<false, false>, 15ul, 16ul> > >, std::vector<long, std::allocator<long> > > const&) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/sink/vtablet_sink.cpp:191
 7# doris::stream_load::VOlapTableSink::send(doris::RuntimeState*, doris::vectorized::Block*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/sink/vtablet_sink.cpp:551
 8# doris::PlanFragmentExecutor::open_vectorized_internal() at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/runtime/plan_fragment_executor.cpp:307
 9# doris::PlanFragmentExecutor::open() at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/runtime/plan_fragment_executor.cpp:247
10# doris::FragmentExecState::execute() at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/runtime/fragment_mgr.cpp:246
11# doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/runtime/fragment_mgr.cpp:499
12# doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}::operator()() const at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/runtime/fragment_mgr.cpp:707
13# void std::__invoke_impl<void, doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}&>(std::__invoke_other, doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:61
14# std::enable_if<is_invocable_r_v<void, doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}&>, void>::type std::__invoke_r<void, doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}&>(doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:117
15# std::_Function_handler<void (), doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}>::_M_invoke(std::_Any_data const&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_function.h:292
16# std::function<void ()>::operator()() const at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_function.h:560
17# doris::FunctionRunnable::run() at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/util/threadpool.cpp:45
18# doris::ThreadPool::dispatch_thread() at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/util/threadpool.cpp:548
19# void std::__invoke_impl<void, void (doris::ThreadPool::*&)(), doris::ThreadPool*&>(std::__invoke_memfun_deref, void (doris::ThreadPool::*&)(), doris::ThreadPool*&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:74
20# std::__invoke_result<void (doris::ThreadPool::*&)(), doris::ThreadPool*&>::type std::__invoke<void (doris::ThreadPool::*&)(), doris::ThreadPool*&>(void (doris::ThreadPool::*&)(), doris::ThreadPool*&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:97
21# void std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>::__call<void, , 0ul>(std::tuple<>&&, std::_Index_tuple<0ul>) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/functional:422
22# void std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>::operator()<, void>() at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/functional:505
23# void std::__invoke_impl<void, std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>&>(std::__invoke_other, std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:61
24# std::enable_if<is_invocable_r_v<void, std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>&>, void>::type std::__invoke_r<void, std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>&>(std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()>&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:117
25# std::_Function_handler<void (), std::_Bind<void (doris::ThreadPool::*(doris::ThreadPool*))()> >::_M_invoke(std::_Any_data const&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_function.h:292
26# std::function<void ()>::operator()() const at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_function.h:560
27# doris::Thread::supervise_thread(void*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/util/thread.cpp:426
28# start_thread in /lib64/libpthread.so.0
29# __clone in /lib64/libc.so.6
```

### What You Expected?

loaded

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by spihiker** [comment]
I have the same problem. All three be nodes core dump
My doris version:1.1.1

*** Aborted at 1662540685 (unix time) try "date -d @1662540685" if you are using GNU date ***
*** SIGSEGV address not mapped to object (@0x0) received by PID 107324 (TID 0x7f625f379700) from PID 0; stack trace: ***
 0# 0x0000560085EF9768 in /ssd/app/doris-1.1.1/be/lib/doris_be
 1# 0x00007F62A39B1400 in /lib64/libc.so.6
 2# doris::vectorized::VSortNode::merge_sort_read(doris::RuntimeState*, doris::vectorized::Block*, bool*) in /ssd/app/doris-1.1.1/be/lib/doris_be
 3# doris::vectorized::VSortNode::get_next(doris::RuntimeState*, doris::vectorized::Block*, bool*) in /ssd/app/doris-1.1.1/be/lib/doris_be
 4# doris::PlanFragmentExecutor::get_vectorized_internal(doris::vectorized::Block**) in /ssd/app/doris-1.1.1/be/lib/doris_be
 5# doris::PlanFragmentExecutor::open_vectorized_internal() in /ssd/app/doris-1.1.1/be/lib/doris_be
 6# doris::PlanFragmentExecutor::open() in /ssd/app/doris-1.1.1/be/lib/doris_be
 7# doris::FragmentExecState::execute() in /ssd/app/doris-1.1.1/be/lib/doris_be
 8# doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>) in /ssd/app/doris-1.1.1/be/lib/doris_be
 9# std::_Function_handler<void (), std::_Bind_result<void, void (doris::FragmentMgr::*(doris::FragmentMgr*, std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>))(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>)> >::_M_invoke(std::_Any_data const&) in /ssd/app/doris-1.1.1/be/lib/doris_be
10# doris::ThreadPool::dispatch_thread() in /ssd/app/doris-1.1.1/be/lib/doris_be
11# doris::Thread::supervise_thread(void*) in /ssd/app/doris-1.1.1/be/lib/doris_be
12# start_thread in /lib64/libpthread.so.0
13# clone in /lib64/libc.so.6

**Comment by carlvinhust2012** [comment]
I also have the problem.
https://github.com/apache/doris/issues/12453

**Comment by yangzhg** [comment]
may fix by #12375
